client/app/dedicatedUniverseComponents/price/price.filter.js

```diff
@@ -40,16 +50,18 @@ export default /* @ngInject */ ($translate) => {
     NL: frenchTouch,
     PL: frenchTouch,
     PT: frenchTouch,
-    SG: usTouch,
+    SG: asiaTouch,
     TN: frenchTouch,
     US: usTouch,
   };
 
   function format(price, paramCountry) {
     const country = (angular.isString(paramCountry) && paramCountry) || 'FR';
     const taxes = showTaxes[country];
-
     if (price.withTax.value !== 0) {
+      if (taxes.withGST) {
```

Comment:
Maybe you could import the template from a `html` file instead:

```import template from "./price.html"```

Reply:
There are multiple templates with if condition in this one. Dont think importing a template will help in this case.

Reply:
Separation of concerns is still appreciate but you're right it could be addressed in a separate pull request since this one has already the `quality check ok` label.

Thanks!